src/data/json-update-builder.js

```diff
@@ -0,0 +1,86 @@
+const OPERATIONS = {
+  SET         : 'JSON_SET',
+  INSERT      : 'JSON_INSERT',
+  REPLACE     : 'JSON_REPLACE',
+  REMOVE      : 'JSON_REMOVE',
+  ARRAY_APPEND: 'JSON_ARRAY_APPEND',
+  ARRAY_INSERT: 'JSON_ARRAY_INSERT'
+}
+
+const OPERATION_FIELD_NAME = '___operation'
+const ARGS_FIELD_NAME = 'args'
+
+export default class JSONUpdateBuilder {
+  constructor() {
+    this.operationName = null
+    this.args = null
+  }
+
+  static initContext(operationName, args) {
```

Comment:
if the class will be able to accept  `operationName, args` in the constructor I assume we would remove the `initContextMethod`

Reply:
+